di-ipv-orchestrator-stub/src/main/java/uk/gov/di/ipv/stub/orc/models/InheritedIdentityJWT.java

```diff
@@ -0,0 +1,27 @@
+package uk.gov.di.ipv.stub.orc.models;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.annotation.JsonValue;
+
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.Map;
+
+@JsonInclude(JsonInclude.Include.NON_NULL)
+public class InheritedIdentityJWT {
+    private String[] value;
```

Comment:
Sorry Talat, it looks like this was updated earlier today to be `values` (not `value`). I know this must be frustrating!

https://github.com/govuk-one-login/architecture/pull/554/files